site/pages/api/publish.api.ts

```diff
@@ -103,6 +109,27 @@ const publish = async (req: NextApiRequest, res: NextApiResponse) => {
 
         cleardownCookies(req, res);
 
+        if (session.isOrgStaff && !template) {
```

Comment:
Could we refactor this to a function that can be re-used in publish-edit flow?

Reply:
refactored to create a separate function sendDisruptionApprovalEmail()